superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -501,22 +529,79 @@ export default class ResultSet extends React.PureComponent<
     return <div />;
   }
 
+  onAlertClose = () => {
+    this.setState({ alertIsOpen: false });
+  };
+
   renderRowsReturned() {
-    const { results, rows, queryLimit } = this.props.query;
-    const limitReached = results?.displayLimitReached;
+    const { results, rows, queryLimit, limitingFactor } = this.props.query;
+    let limitMessage;
+    const isAdmin = this.props.user?.roles.hasOwnProperty('Admin');
+    const adminWarning = isAdmin
+      ? ' by the configuration DISPLAY_MAX_ROWS'
+      : null;
```

Comment:
if you only have one option, it's better to use && `const adminWarning = isAdmin && ' by the....'. it will be falsy otherwise, so setting it to null is redundant, unless you explicitly need null.

Reply:
Yeah I explicitly need null here otherwise it returns undefined in the Alert 

```
 description={t(
              `The number of results displayed is limited to %s%s. Please add
              additional limits/filters or download to csv to see more rows up to
              the %s limit. %s`,
              rows,
              adminWarning,
              queryLimit,
            )}
```

Reply:
My original method was two separate Alerts one for admin and one for non-admin. This saves space. Happy to go back to the other method though if the construction with && is DRY-er

Reply:
oh, ok.. then can you do an empty string?

Reply:
yeah that works, ty.